acq4/devices/PatchPipette/testpulse.py

```diff
@@ -353,22 +347,17 @@ def analysis(self):
                 Q = (pulseData - yoffset).sum() * dt
                 Rin = analysis['steadyStateResistance']
                 Vc = params['amplitude']
-                Rs_denom = (Q * Rin + tau * Vc)
+                Rs = analysis['peakResistance']
+                Rm = Rin - Rs
+                Rs_denom = (Rm**2 * abs(Vc))
                 if Rs_denom != 0.0:
-                    # Rs = (Rin * tau * Vc) / Rs_denom
-                    Rs = analysis['peakResistance']
-                    Rm = Rin - Rs
-                    Cm = (Rin**2 * Q) / (Rm**2 * abs(Vc))
+                    analysis['capacitance'] = (Rin**2 * Q) / Rs_denom
                 else:
-                    Rs = 0
-                    Rm = 0
-                    Cm = 0
-                analysis['capacitance'] = Cm
+                    analysis['capacitance'] = 0
+            elif analysis['steadyStateResistance'] > 0:
+                analysis['capacitance'] = tau / analysis['steadyStateResistance']
             else:
-                if analysis['steadyStateResistance'] > 0:
-                    analysis['capacitance'] = tau / analysis['steadyStateResistance']
-                else:
-                    analysis['capacitance'] = np.nan
+                analysis['capacitance'] = np.nan
```

Comment:
The original structure had the if-block for VC and the else-block for IC; I think it's more readable as it was?